fearless/Modules/Pincode/CheckPincode/CheckPincodePresenter.swift

```diff
@@ -43,7 +43,7 @@ extension CheckPincodePresenter: LocalAuthInteractorOutputProtocol {
         }
     }
 
-    func didChangeState(from _: LocalAuthInteractor.LocalAuthState) {}
+    func didChangeState(to _: LocalAuthInteractor.LocalAuthState) {}
```

Comment:
unused parameter

Reply:
yes, this is LocalAuthInteractorOutputProtocol